src/bindings/python/tests/test_runtime/test_compiled_model.py

```diff
@@ -151,200 +108,127 @@ def test_output_set_friendly_name(device):
+                    reason=f"Cannot run test on device {os.environ.get('TEST_DEVICE')}, Plugin specific test")
 def test_infer_tensor_numpy_shared_memory(device):
-    core = Core()
-    model = core.read_model(model=test_net_xml, weights=test_net_bin)
-    img = generate_image()
+    compiled_model, img = generate_model_and_image(device)
+
     img = np.ascontiguousarray(img)
     tensor = Tensor(img, shared_memory=True)
-    compiled_model = core.compile_model(model, device)
     res_tensor = compiled_model.infer_new_request({"data": tensor})
     res_img = compiled_model.infer_new_request({"data": img})
-    assert np.argmax(res_tensor[list(res_tensor)[0]]) == 9
+    assert np.argmax(res_tensor[list(res_tensor)[0]]) == 531
     assert np.argmax(res_tensor[list(res_tensor)[0]]) == np.argmax(res_img[list(res_img)[0]])
 
 
 def test_infer_new_request_wrong_port_name(device):
-    core = Core()
-    model = core.read_model(model=test_net_xml, weights=test_net_bin)
-    img = generate_image()
+    compiled_model, img = generate_model_and_image(device)
+
     tensor = Tensor(img)
-    compiled_model = core.compile_model(model, device)
     with pytest.raises(RuntimeError) as e:
         compiled_model.infer_new_request({"_data_": tensor})
     assert "Check" in str(e.value)
 
 
 def test_infer_tensor_wrong_input_data(device):
-    core = Core()
-    model = core.read_model(model=test_net_xml, weights=test_net_bin)
-    img = generate_image()
+    compiled_model, img = generate_model_and_image(device)
+
     img = np.ascontiguousarray(img)
     tensor = Tensor(img, shared_memory=True)
-    compiled_model = core.compile_model(model, device)
     with pytest.raises(TypeError) as e:
         compiled_model.infer_new_request({0.: tensor})
     assert "Incompatible key type for input: 0.0" in str(e.value)
 
 
-def test_infer_numpy_model_from_buffer(device):
-    core = Core()
-    with open(test_net_bin, "rb") as f:
-        weights = f.read()
-    with open(test_net_xml, "rb") as f:
-        xml = f.read()
-    model = core.read_model(model=xml, weights=weights)
-    img = generate_image()
-    compiled_model = core.compile_model(model, device)
-    res = compiled_model.infer_new_request({"data": img})
-    assert np.argmax(res[list(res)[0]]) == 9
-
-
-def test_infer_tensor_model_from_buffer(device):
-    core = Core()
-    with open(test_net_bin, "rb") as f:
-        weights = f.read()
-    with open(test_net_xml, "rb") as f:
-        xml = f.read()
-    model = core.read_model(model=xml, weights=weights)
-    img = generate_image()
-    tensor = Tensor(img)
-    compiled_model = core.compile_model(model, device)
-    res = compiled_model.infer_new_request({"data": tensor})
-    assert np.argmax(res[list(res)[0]]) == 9
-
-
+@pytest.mark.skipif(os.environ.get("TEST_DEVICE", "CPU") != "CPU",
+                    reason=f"Cannot run test on device {os.environ.get('TEST_DEVICE')}, Plugin specific test")
```

Comment:
the same

Reply:
double checked with GPU plugin, this skip may be removed.